spec/features/idv/end_to_end_idv_spec.rb

```diff
@@ -50,12 +55,72 @@
     validate_personal_key_page
     acknowledge_and_confirm_personal_key
 
-    validate_idv_completed_page
+    validate_idv_completed_page(user)
     click_agree_and_continue
 
     validate_return_to_sp
   end
 
+  context 'with an sp that allows in person proofing' do
+    before do
+      allow(IdentityConfig.store).to receive(:in_person_proofing_enabled).and_return(true)
+
+      ServiceProvider.find_by(issuer: service_provider_issuer(sp)).
+        update(in_person_proofing_enabled: true)
+    end
+
+    scenario 'In person proofing verify by mail', allow_browser_log: true do
+      visit_idp_from_sp_with_ial2(sp)
+      user = sign_up_and_2fa_ial1_user
+
+      begin_in_person_proofing
+      complete_all_in_person_proofing_steps(user)
+
+      enter_gpo_flow
+      gpo_step
+
+      complete_review_step(user)
+
+      validate_come_back_later_page
+      complete_come_back_later
+      validate_return_to_sp
+
+      visit sign_out_url
+
+      visit_idp_from_sp_with_ial2(sp)
+
+      sign_in_live_with_2fa(user)
+
+      complete_gpo_verification(user)
+      expect(user.identity_verified?).to be(false)
+
+      acknowledge_and_confirm_personal_key
+
+      # click return to sp link
```

Comment:
should we make this into a method? something like `click_return_to_sp_from_ipp`?

Reply:
Turns out there was a helper. I changed its name to make it more findable.